### ISSUE TITLE
Display placeholders for autoscoped-constructor arguments

### ISSUE THREAD

**Issue by kazcw** [enso-org/enso#9635]
Currently `WidgetFunction` uses `WidgetInput.isFunctionCall` to recognize function calls or *potential* function calls (e.g. `Ast.Ident`). It should accept `Ast.AutoscopedIdentitifier`--or maybe we could eliminate this check and use `WidgetFunction` for any `Ast` that we have a `getMethodCallInfo` result for, and let the engine decide what's a function?

This is necessary to fix: https://github.com/enso-org/enso/issues/9614#issuecomment-2036639574

**Comment by farmaazon** [comment]
Refinement notes:
1. The AST "guards" should stay to avoid unnecessary `MethodCallIInfo` reads. We should just add one more case there.
2. Extend E2E tests to check if autoscoped constructors works properly (have arguments).

**Comment by JaroslavTulach** [comment]
I am trying to verify that everything works, but no luck. I don't think this is fixed. To reproduce my steps apply following diff:
```diff
enso.ide$ git diff
diff --git distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso
index fe6731efa2..e10d1ed703 100644
--- distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso
+++ distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso
@@ -23,7 +23,7 @@ make_aggregate_column_selector table display=Display.Always =
     col_names_selector = make_column_name_selector table display=Display.Always add_expression=True
     column_widget = ["column", col_names_selector]
 
-    fqn = Meta.get_qualified_type_name Aggregate_Column
+    fqn = "."
     count = Option "Count" fqn+".Count"
 
     ## Currently can't support nested vector editors so using single picker
```
rebuild and run the project manager using following command:
```bash
enso$ ./run backend sbt -- runProjectManagerDistribution
 INFO ide_ci::program::command: sbtℹ️ Enso Project Manager
 INFO ide_ci::program::command: sbtℹ️ Version:    2024.1.1-dev
 INFO ide_ci::program::command: sbtℹ️ Built with: scala-2.13.11 for GraalVM 21.0.2
 INFO ide_ci::program::command: sbtℹ️ Built from: develop* @ 447f4b5ac6a4e757db41c7e0d3d9252c8153ac7c
 INFO ide_ci::program::command: sbtℹ️ Built on:   Linux (amd64)
 INFO ide_ci::program::command: sbtℹ️ 
 INFO ide_ci::program::command: sbtℹ️ [INFO] [2024-04-29T08:12:10+02:00] [org.enso.projectmanager.boot.ProjectManager$] Started server at 127.0.0.1:30535, press enter to kill server
```
Once the project manager is started, run the IDE in another terminal
```bash
enso$ npm --workspace=enso-gui2 run dev
```
When I try to use `aggregate` function, I don't see a choice box on the `..Sum` to let me select name of a column:

![No column drop down](https://github.com/enso-org/enso/assets/26887752/b93839e8-4d90-4ac9-8aae-782c1883d615)

that's different to what I see when running with the patched `distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso` file:

![Column drop down](https://github.com/enso-org/enso/assets/26887752/895272f5-2ba7-4d8e-a41b-d35603b5a200)

With fully qualified name of the constructor, there is a drop down to select a column. Such a drop down isn't visible when working with autoscoped constructor.

I tried to verify this issue multiple times during last week, but the result is always the same. Reopening. @vitvakatu, please try my use-case and sorry, if it works for you.
